tests/acceptance/project-checkout-test.js

```diff
@@ -5,7 +5,7 @@ import Mirage from 'ember-cli-mirage';
 
 import { authenticateSession } from 'code-corps-ember/tests/helpers/ember-simple-auth';
 import { getFlashMessageCount } from 'code-corps-ember/tests/helpers/flash-message';
-import projectDonatePage from '../pages/project/donate';
```

Comment:
We'll need a page for the new `projectDonatePage`, borrowing probably from the component (or using it directly, I think).